src/cdp/cdpConnection.ts

```diff
@@ -78,15 +79,15 @@ export class CdpConnection {
     return cdpClient;
   }
 
-  sendCommand(
-    method: string,
-    params: object | undefined,
+  sendCommand<CdpMethod extends keyof ProtocolMapping.Commands>(
+    method: CdpMethod,
+    params: ProtocolMapping.Commands[CdpMethod]['paramsType'][0] | undefined,
     sessionId: string | null
   ): Promise<object> {
     return new Promise((resolve, reject) => {
       const id = this.#nextId++;
       this.#commandCallbacks.set(id, {resolve, reject});
-      const messageObj: CdpMessage = {id, method, params};
+      const messageObj: CdpMessage<typeof method> = {id, method, params};
```

Comment:
Can we use just `CdpMethod` method here?

Reply:
Good catch. Done!